Makefile

```diff
@@ -30,7 +30,7 @@ generate-protobuf:
                 protoc -I "$(GOPATH)/src/" -I '$(REPO_ROOT)/client/api' "$$FILE" --go_out=plugins="grpc:$(GOPATH)/src"; \
         } ; \
         export -f generate_protobuf_for; \
-        find '$(REPO_ROOT)' -name '*.proto' | sed -e "s|$(GOPATH)/src/||g" | xargs -n1 $(SHELL) -c 'generate_protobuf_for "$$0"'
+        find '$(REPO_ROOT)' -name '*.proto' | sed -e "s|$(GOPATH)/src/||g" | xargs -n1 '$(SHELL)' -c 'generate_protobuf_for "$$0"'
```

Comment:
Are the `''`s really necessary?

Reply:
in my case they were because of spaces in the path